### ISSUE TITLE
[BUG] - Can't build a nextui project and freezes saying "Creating an optimized production build..."

### ISSUE THREAD

**Issue by Melak12** [nextui-org/nextui#848]
I have just created a new project with Next js and NextUI. After making the initial setup, I try to build the project but it freezes saying ' **Creating an optimized production build ...'.** . This happens after adding NextUI to the project. I appreciate for a quick fix. 
I am using: 

- Node v18.5.0
- Next v13.0.0
- "@nextui-org/react": "^1.0.0-beta.10",

Here is the __app.js_: 
```
import { NextUIProvider } from '@nextui-org/react';
import '../styles/globals.css'

function MyApp({ Component, pageProps }) {
  return (
    <NextUIProvider>
      <Component {...pageProps} />
    </NextUIProvider>
  );
}

export default MyApp
```

Here is __document.js_ : 
```
import React from 'react';
import Document, { Html, Head, Main, NextScript } from 'next/document';
import { CssBaseline } from '@nextui-org/react';

class MyDocument extends Document {
  static async getInitialProps(ctx) {
    const initialProps = await Document.getInitialProps(ctx);
    return {
      ...initialProps,
      styles: React.Children.toArray([initialProps.styles])
    };
  }

  render() {
    return (
      <Html lang="en">
        <Head>{CssBaseline.flush()}</Head>
        <body>
          <Main />
          <NextScript />
        </body>
      </Html>
    );
  }
}

export default MyDocument;
```

Here is _package.json_ : 

```
{
  "name": "my-next",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "lint": "next lint"
  },
  "dependencies": {
    "@nextui-org/react": "^1.0.0-beta.10",
    "next": "13.0.0",
    "react": "18.2.0",
    "react-dom": "18.2.0"
  },
  "devDependencies": {
    "eslint": "8.26.0",
    "eslint-config-next": "13.0.0"
  }
}
```

### Steps to Reproduce the Bug or Issue

1. Create a next js project using npx create-next-app@latest
2. Install Next UI: npm i @nextui-org/react
3. Build the project: npm run build

### Expected behavior

As a User, I was expecting successful build artifacts. 

### Screenshots or Videos

![image](https://user-images.githubusercontent.com/45941116/198552925-a648b6be-f6c4-404f-8b96-dec0cd7dbcf0.png)

### Operating System Version

Linux

### Browser

Chrome

**Comment by Melak12** [comment]
**Update:**
After setting swcMinify: false in _next.config.js_ file, it works. But we need to figure out why?

```
/** @type {import('next').NextConfig} */
const nextConfig = {
  reactStrictMode: true,
  swcMinify: false,
}

module.exports = nextConfig
```

**Comment by TedDDDao** [comment]
Thank you, I'm having the same problem and setting `swcMinify: false,` (temporarily) fixed it for me, too.

`npx nextjs build` hangs on "Creating an optimized production build..." even when the only page imports ANY nextui component, using the same dependencies as OP.

**Comment by tianenpang** [comment]
Hi guys, it's been fixed in [next@13.0.3-canary.0](https://github.com/vercel/next.js/releases/tag/v13.0.3-canary.0), install `next@canary` and give it a try.

*The issue can be closed*
